frontend/src/components/TodoListItem.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconDelete } from "./Icons/IconDelete";
+export default function TodoListItem(props: any) {
```

Comment:
Assign a proper type to `props` here as well. Also add an empty line after the `IconDelete` import statement.